postgres_composite_types/__init__.py

```diff
@@ -165,6 +147,9 @@ def database_forwards(self, app_label, schema_editor,
             "AS (%s)" % ', '.join(fields),
         )))
 
+        composite_type_created.send(self.Meta.model,
```

Comment:
Yes. This is required when the types do not yet exist in the database, such as before the migration has run.